sdk/ml/azure-ai-ml/azure/ai/ml/entities/_deployment/request.py

```diff
@@ -0,0 +1,25 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+from typing import Dict
+
+from azure.ai.ml._schema._deployment.online.request_schema import RequestSchema
+from azure.ai.ml.constants._common import BASE_PATH_CONTEXT_KEY
+
+
+class Request:
```

Comment:
Do the other parts of our SDK have very generic class names (differentiated by the package/folder names?) 
Else, may be a small improvement to call it DeploymentRequest and DeploymentResponse. 

Just so we don't have to ask our users to write code like
`from azure.ai.ml.entities import Response as DeployResponse`

Not a blocking comment, just a thought

Reply:
I agree we should reconsider naming it "Request" and the same for "Response", hard part is I don't have the visibility of this feature fully, so it will be useful to add tests cases with yml, so that the review becomes more comprehensive.

Reply:
I haven't found any classes as generic as "response" and "request" but I have found that the class names usually match the expected name of the yaml properties. However, if you believe it would significantly improve code I can make the change.